### PR TITLE
Add bot feature to rename new members

### PR DESCRIPTION
**Pull request by ocanash1**
to run locally and test, you can create a new discord bot, get the token, update the .env file, and run the main.py file.

currently hosting a version of this bot on replit.com but not sure how the DAO wants to deploy this? aws? heroku? will we have a ci/cd pipeline?

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+TOKEN=
```
Comment (CryptoSpaces) [inline .env]: Could we add a `.gitignore` to the project to exclude these `.env` files

File: main.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+@bot.event
+async def on_member_join(member):
+  prev_name = member.name
+  await member.edit(nick=DEFAULT_NAME)
+
+  for c in member.guild.channels:
+    if "noob-chat" in str(c):
+      await c.send("Welcome to the pizza party, {0}! [formally {1}]".format(member.nick, prev_name))
+
+@bot.command(name="avatars")
```
Comment (CryptoSpaces) [inline main.py]: What is this avatar command for? Why would we want to extract avatar images?

Comment (ocanash1) [reply main.py]: there was talk about exporting all the prof pics from our discord members to use in an infographic. thinking on it more, we prob don't need this as a command. leaves it open for spammers to spam. could just make it a private script that one of us can run to download the files locally if/when needed. what do you think?

**Review by CryptoSpaces — APPROVED** [review]

**Comment by ocanash1** [comment]
added additional changes to the same PR since we still haven't merged. will reach out to Don Luv about deploying on azure

File: main.py [hunk 3]
```diff
@@ -36,47 +56,48 @@ async def send_files(ctx, files):
...
 
 @bot.event
 async def on_member_join(member):
   prev_name = member.name
   await member.edit(nick=DEFAULT_NAME)
 
+  # sometimes member.nick is None so maybe sleep 
+  # to ensure changes are committed? todo: investigate this more
+  time.sleep(1) 
   for c in member.guild.channels:
```
Comment (CryptoSpaces) [inline main.py]: Could rename `c` to `channel` to be more explicit

**Review by CryptoSpaces — APPROVED** [review]

**Review by CryptoSpaces — APPROVED** [review]

**Comment by CryptoSpaces** [comment]
@ocanash1 can we merge this?

**Review by CryptoSpaces — APPROVED** [review]

**Comment by ocanash1** [comment]
> This looks good to me.
> 
> Is "toitles" (new line 75) intentionally spelled that way? "to pick ur toitles then report here for ur mafia name" If so, then it's fine, just thought I would mention it in case it wasn't intentional.

haha yeah i just copied a few messages from snax and used those verbatim. i'm not as creative 🙃

**Pull request merged**